src/Command/InstallCommand.php

```diff
@@ -64,15 +64,14 @@ protected function configure()
      */
     protected function executeLocked(InputInterface $input, OutputInterface $output)
     {
-        $fs      = new Filesystem();
-        $rootDir = dirname($this->getContainer()->getParameter('kernel.root_dir'));
+        $fs = new Filesystem();
 
         foreach ($this->emptyDirs as $path) {
-            $this->addEmptyDir("$rootDir/$path", $fs, $output);
+            $this->addEmptyDir(getcwd() . "/$path", $fs, $output);
```

Comment:
Why did you change this to `getcwd()`? Now the script can only be run from the installation root directory, which is wrong. It is ok to use `getcwd()` in a Composer script handler, because Composer **always** runs in the installation root directory. But `app/console` does not, so please revert this.